@xen-orchestra/vmware-explorer/VhdEsxiRaw.mjs

```diff
@@ -204,6 +207,9 @@ export default class VhdEsxiRaw extends VhdAbstract {
       return stream
     })
   }
+  async dispose() {
+    await this.#stream?.destroy()
+  }
 }
```

Comment:
I you are not using this method directly, I think it would be better to remove inline this method in `open()`.

Also, that would better fit the Disposable API.

Reply:
I don't understand how the static method open will be able to access the private `#stream` property

Reply:
IIRC, every code defined inside the class can access private fields, regardless whether they are static.

Reply:
I didn't know this, thanks. But I prefer not to use this counter intuitive feature.

Reply:
It's not counter intuitive because you are not used to it, some languages implement visibility at the class level and other at the instance level.

But you don't have to use it if you don't want to.